### ISSUE TITLE
New release 0.0.3 on pypi?

### ISSUE THREAD

**Issue by mgazic** [blauret/pyG5#11]
I've been using pyG5 for a while, and I noticed some months back that it didn't work with the latest python 3.10 version of python, because of changes in Python not interacting well with pyqt5 (something about auto conversion between float and int, that affected many other pyqt5 projects, not just yours).  I cloned your repo and discovered the latest code works under python 3.10, so thank you very much for your recent work that fixed that!

However, there hasn't been a new release since 0.0.2 in May of 2022.  Any chance you'll do a new release soon and push 0.0.3 to pypi?  If you do that, then I can simply `pip install pyg5` to get a version that works for me.  Otherwise I need to clone your repo and do `pip install --editable .`

Thanks for creating this cool project in the first place, and making it open source!

**Comment by blauret** [comment]
Thanks for your input. Glad to see users!!!

I've been running the secondary_screen branch in my setup recently. It's adding an advisory panel and some controls specifically for the C172. See picture

<img width="912" alt="image" src="https://user-images.githubusercontent.com/2811733/202651391-825c96e6-d4c0-41b8-bd03-4ab985dc9384.png">

I guess this is not for everybody and I need to add options to start with or without. That's what holding me for a new release. I also made visual improvements and it's very close to a real G5 now (I'm doing my flight training with this setup).

I also saw [this fork|https://github.com/michaelplomer/pyG5] has support for Qt6. Which I would like to merge too.

A lot of words to say, it will come :)
